ros2test/package.xml

```diff
@@ -0,0 +1,26 @@
+<?xml version="1.0"?>
+<?xml-model
+  href="http://download.ros.org/schema/package_format2.xsd"
+  schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>ros2test</name>
+  <version>0.8.1</version>
```

Comment:
Any particular reason for starting at this version?  I don't think it matters where you start, but I was just curious.

Reply:
Just matching the version of most of the other `launch` packages. Any other way you'd rather go about it? I don't have a strong preference and I can't tell if we have a stricter policy on this regard.

Reply:
Since both are in separate repositories they will level independently. So there is no reason to align them now. Please start this repo with version 0.1.0.

Reply:
Fair enough.